core-primitives/stf-executor/src/executor_tests.rs

```diff
@@ -92,32 +88,31 @@ pub fn propose_state_update_executes_only_one_trusted_call_given_not_enough_time
 	let mrenclave = ocall_api.get_mrenclave_of_self().unwrap().m;
 	let (_, shard) = init_state_and_shard_with_state_handler(state_handler.as_ref());
 	let sender = endowed_account();
-	let signed_call_1 = TrustedCall::balance_transfer(
-		sender.public().into(),
-		sender.public().into(),
-		42,
-	)
-	.sign(&sender.clone().into(), 0, &mrenclave, &shard);
+	let signed_call_1 = TrustedCallMock::waste_time_ms(sender.public().into(), 10).sign(
```

Comment:
the test before was brittle as it depended on the performance of the testing machine. now we can cause precise durations